cfg/config.go

```diff
@@ -3,11 +3,84 @@
 // that will be retrieved from  .env or ./cfg/config.yml
 package cfg
 
+import (
+	"bufio"
+	"log"
+	"os"
+	"strings"
+)
+
+// Load configs from a env file & sets them in environment variables .
```

Comment:
In this comment you have a space before dot - please remove it.
Another point - here you have a dot but in other comment - not. Please, try to use the same approach in all places.

Code should look the same.

Reply:
Fixed